front/src/app/components/admin/conventions/BroadcastAgainButton.tsx

```diff
@@ -30,6 +37,16 @@ export const BroadcastAgainButton = ({
   const [isModalButtonDisabled, setIsModalButtonDisabled] = useState(
     disabled ?? false,
   );
+  const feedbacks = useAppSelector(feedbacksSelectors.feedbacks);
+
+  const onCloseBroadcastFeedbackModal = () => {
```

Comment:
```suggestion
  const closeBroadcastFeedbackModal = () => {
```
vu que ça déclenche le close, ça peut pas être onClose

Reply:
je marque en résolu car une autre solution a finalement été implémentée